### PR TITLE
Ios sdk - update to work with SDK 3

### PR DESCRIPTION
**Pull request by MongoCaleb**

### CODE REVIEW

File: todo/ios/MongoDBSample/Classes/Controllers/Authentication/AuthenticationViewController.swift [hunk 1]
```diff
@@ -76,12 +82,13 @@ class AuthenticationViewController: UIViewController, GIDSignInUIDelegate, GIDSi
                 authAvailable = true
                 
                 var configureError: NSError?
-                GGLContext.sharedInstance().configureWithError(&configureError)
+                //GGLContext.sharedInstance().configureWithError(&configureError)
```
Comment (adamchel) [inline todo/ios/MongoDBSample/Classes/Controllers/Authentication/AuthenticationViewController.swift]: should this be commented out?

File: todo/ios/MongoDBSample/Resources/Stitch-Info.plist [hunk 2]
```diff
@@ -3,6 +3,6 @@
 <plist version="1.0">
 <dict>
 	<key>APP_ID</key>
-	<string>YOUR-APP_ID</string>
+	<string>todo_web_app-kkqke</string>
```
Comment (adamchel) [inline todo/ios/MongoDBSample/Resources/Stitch-Info.plist]: Should be a placeholder

File: todo/ios/Podfile [hunk 3]
```diff
@@ -3,11 +3,12 @@ platform :ios, '9.0'
 target 'MongoDBSample' do
   use_frameworks!
 
-  pod 'Google/SignIn'
+  pod 'Firebase/Core'
```
Comment (adamchel) [inline todo/ios/Podfile]: Is this necessary? See what happens if you remove this, run `pod install` and try to build again.

File: todo/ios/MongoDBSample/Classes/Controllers/Authentication/AuthenticationViewController.swift (hunk 1 above)
Comment (MongoCaleb) [reply todo/ios/MongoDBSample/Classes/Controllers/Authentication/AuthenticationViewController.swift]: Looks like my latest changes didn't get pushed. :(

**Pull request merged**